### ISSUE TITLE
 Installing on openSUSE Leap 15.0

### ISSUE THREAD

**Issue by janot** [corrad1nho/qomui#24]
I've kinda installed it, but not able to add new provider manually

Sometimes it just crashes on pressing + Add Folder
When it allows me to choose wireguard config file it says:
```
Qomui: Importing Foobar successful
List of available servers updated
```
but server config isn't actually being created

I'm not 100% I've installed all required dependencies. I had some difficulties with that e.g. wasn't able to find python-dbus.mainloop.pyqt5

**Comment by corrad1nho** [comment]
I have never tested Qomui on OpenSuse, but I will try it. Which version of Qomui are you using? When it says " Importing Foobar successful" the main dependencies are definitively installed otherwise you wouldn't get that far. Have a look at the log and see if resolving hostnames for your servers failed as Qomui will not add those servers as the firewall relies on resolved hostnames. Can you tell me the provider of those WireGuard config files?

**Comment by corrad1nho** [comment]
I tested on OpenSuse Leap 15.0 fresh install: I had no issues adding WireGuard configs for Mullvad and AzireVPN. I encountered one bug, though: If you leave the "Specify provider" field empty, Qomui crashes - a little bug I overlooked when I switched most messages to libnotify. This will be fixed in the next release. 

Couple of things to try: Have a look at ~/.qomui/server.json - does this file exist? If yes, is it empty? I'd also be happy to test a specific config file if you can provide one (be sure to replace your keys with some bogus text or leave empty). 

I'll also look into supporting OpenSuse properly - OBS sounds quite interesting.

**Comment by corrad1nho** [comment]
Update: I think I found the issue - most likely resolving the endpoint failed - I just forgot to include a log message in that case. Look in your config file and try resolving the endpoint manually with dig:
`dig $wireguard-server`

If it fails, check if "bind-utils" are installed.

**Comment by janot** [comment]
That's what inside ~/.qomui/server.json: 
`{}`

Here's output of `dig $wireguard-server`               

```
; <<>> DiG 9.11.2 <<>> deletedip
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 58770
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;deletedip                  IN      A

;; AUTHORITY SECTION:
.                       3600    IN      SOA     a.root-servers.net. nstld.verisign-grs.com. 2018082800 1800 900 604800 86400

;; Query time: 48 msec
;; SERVER: 208.67.220.220#53(208.67.220.220)
;; WHEN: Tue Aug 28 15:57:53 +03 2018
;; MSG SIZE  rcvd: 116
```
Here's config file:
```
# "name" - Streisand WireGuard Client Profile
[Interface]
Address = 10.192.122.2/32
# The use of DNS below effectively expands to:
#   PostUp = echo nameserver 10.192.122.1 | resolvconf -a tun.%i -m 0 -x
#   PostDown = resolvconf -d tun.%i
# If the use of resolvconf is not desirable, simply remove the DNS line
# and use a variant of the PostUp/PostDown lines above.
# The IP address of the DNS server that is available via the encrypted
# WireGuard interface is 10.192.122.1.
DNS = 10.192.122.1
PrivateKey = deleted

[Peer]
PublicKey = deleted
AllowedIPs = 0.0.0.0/0
Endpoint = deleted:51820
```

**Comment by corrad1nho** [comment]
Can't see anything there that would trip Qomui up. Dig command works, too. Can you try putting the resolved endpoint-ip in the config or is that already the case?

**Comment by janot** [comment]
It's already there. I've deleted it before posting config here
I can send you full config through one-time messages

**Comment by corrad1nho** [comment]
Yeah, that'd be great!

**Comment by janot** [comment]
https://privnote.com/pCYhVlLO#xEOjXa1qv
Tell me when I can disable it, please

**Comment by corrad1nho** [comment]
you can disable it, I got it and checking it now.

**Comment by janot** [comment]
I mean I will re-create whole server, so you won't be able to connect anymore. I suppose you still need it

**Comment by corrad1nho** [comment]
Found the issue, there's a line missing in the import method - funnily, it should work with unresolved hostnames. I will release an update tomorrow. Thanks a lot, this was very valuable as I don't have a whole lot of WireGuard configs to test with!

PS: I did a quick check if I could connect (which was successful) with your config but have since deleted all files.

**Comment by corrad1nho** [comment]
I just released v0.7.3. It should work now.
